os/arch/arm/src/common/up_exit.c

```diff
@@ -167,6 +170,12 @@ void _exit(int status)
 	sched_foreach(_up_dumponexit, NULL);
 #endif
 
+#ifdef CONFIG_ARMV8M_TRUSTZONE
+	struct tcb_s *dtcb = this_task();
+	if (dtcb->secure_handle) {
+		up_free_secure_context(dtcb->secure_handle);
+	}
+#endif
```

Comment:
1. Is this proprietary location? Almost kernel resources are freed at task_exithook function.
2. There is the tcb variable already. Do we need to define and to use new tcb variable?

Reply:
1. I preferred this location as it is arch specific folder and the modification is architecture based, task_exithook is in generic task related location
2. will re-use tcb

Reply:
ok